src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionNavigationController.swift

```diff
@@ -157,6 +157,13 @@ class ExposureSubmissionNavigationController: UINavigationController, UINavigati
 		exposureSubmissionService
 	}
 
+	override func viewDidLayoutSubviews() {
+		super.viewDidLayoutSubviews()
+
+		updateBottomSafeAreaInset()
+		view.layoutIfNeeded()
```

Comment:
Is it good idea to use it here? It could lead to stackoverflow issue in this method.